SAGESharp/SLB/Level/Common.cs

```diff
@@ -0,0 +1,28 @@
+﻿/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+using SAGESharp.IO;
+using System.Collections.Generic;
+
+namespace SAGESharp.SLB.Level
+{
+    public sealed class SPLinePath
```

Comment:
Why the odd capitalization? It's a "SplinePath", right? (https://en.wikipedia.org/wiki/Spline_(mathematics))